### PR TITLE
Task 5

### PR DESCRIPTION
**Pull request by anton-yatskevich**

### CODE REVIEW

File: client/src/constants/MovieSchema.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.objectOf(
```
Comment (AlexeyBarkow) [inline client/src/constants/MovieSchema.js]: You could use `PropTypes.shape`

File: client/src/utils/sortHelper.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+function filterDatesHelper(a, b) {
+    const momentA = moment(a.release_date, 'YYYY-MM-DD');
+    const momentB = moment(b.release_date, 'YYYY-MM-DD');
+
+    return momentB - momentA;
+}
+
+const sortHelper = (movies, sortField) => (
+    [...movies].sort((a, b) => {
```
Comment (AlexeyBarkow) [inline client/src/utils/sortHelper.js:11]: I'd recommend to use `reselect` lib for your data slices. For now your approach recalculates movies array on each action dispatch, which can cause performance issues in a long term

File: client/__snapshots__/index.spec.js.snap [hunk 3]
```diff
@@ -1,3 +1,30 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Root component test should call ReactDOM.render and match snapshot 1`] = `<AppContainer />`;
+exports[`Root component test should call ReactDOM.render and match snapshot 1`] = `
```
Comment (AlexeyBarkow) [inline client/__snapshots__/index.spec.js.snap:3]: Please, update your snapshots (`npm run test -- --updateSnapshot`)

**Review by AlexeyBarkow — CHANGES_REQUESTED** [review]

File: client/src/constants/MovieSchema.js (hunk 1 above)
Comment (anton-yatskevich) [reply client/src/constants/MovieSchema.js]: Updated with `PropTypes.shape`

File: client/src/utils/sortHelper.js (hunk 2 above)
Comment (anton-yatskevich) [reply client/src/utils/sortHelper.js:11]: Selector for sorting movies has been added

File: client/__snapshots__/index.spec.js.snap (hunk 3 above)
Comment (anton-yatskevich) [reply client/__snapshots__/index.spec.js.snap:3]: I've run tests with `--update Snapshot` flag but nothing was changed

**Review by AlexeyBarkow — APPROVED** [review]

**Pull request merged**